site/src/comparison.rs

```diff
@@ -476,55 +476,23 @@ pub fn write_summary_table(
         let value = if count > 0 { calculate() } else { None };
         value
             .map(|value| format!("{value:.1}%"))
-            .unwrap_or_else(|| "N/A".to_string())
+            .unwrap_or_else(|| "".to_string())
     }
 
-    let columns = [
-        "          ", // we want at least 10 spaces to accommodate "count[^1]"
-        "Regressions 😿 <br />(primary)",
-        "Regressions 😿 <br />(secondary)",
-        "Improvements 🎉 <br />(primary)",
-        "Improvements 🎉 <br />(secondary)",
-        "All 😿 🎉 <br />(primary)",
-    ];
-    let counts: Vec<usize> = columns
-        .iter()
-        .map(|s| {
-            // Unicode emojis usually have width equal to ~2 chars (maybe? :) ).
-            let count = s.chars().count();
-            if s.chars().any(|c| !c.is_whitespace()) {
-                count + 1
-            } else {
-                count
-            }
-        })
-        .collect();
-    for column in &columns {
-        write!(result, "| {} ", column).unwrap();
-    }
-    result.push_str("|\n");
-    for &count in &counts {
-        write!(result, "|:{}:", "-".repeat(count)).unwrap();
-    }
-    result.push_str("|\n");
+    // (label, mean, max, count)
+    let mut column_data = vec![];
 
-    let mut render_row = |row: Vec<String>| {
-        debug_assert_eq!(row.len(), columns.len());
-        for (column, &count) in row.into_iter().zip(&counts) {
-            write!(result, "| {:<1$} ", column, count).unwrap();
-        }
-        result.push_str("|\n");
-    };
-    render_row(vec![
-        format!("count{}", if with_footnotes { "[^1]" } else { "" }),
-        primary.num_regressions.to_string(),
-        secondary.num_regressions.to_string(),
-        primary.num_improvements.to_string(),
-        secondary.num_improvements.to_string(),
-        (primary.num_regressions + primary.num_improvements).to_string(),
+    // label
+    column_data.push(vec![
+        "Worse (primary)".to_string(),
```

Comment:
I personally find "Worse" and "Better" to be more confusing. How about "Improve" and "Regress"?

Reply:
I found improvements/regressions fine, but there was some interest in changing them (https://rust-lang.zulipchat.com/#narrow/stream/247081-t-compiler.2Fperformance/topic/Tables.20bad.20for.20TWIR.20format/near/283322938, https://rust-lang.zulipchat.com/#narrow/stream/247081-t-compiler.2Fperformance/topic/Tables.20bad.20for.20TWIR.20format/near/282641528). :man_shrugging: